lib/Doctrine/ORM/Event/OnClearEventArgs.php

```diff
@@ -67,7 +67,7 @@ public function getEntityManager()
     /**
      * Name of the entity class that is cleared, or empty if all are cleared.
      *
-     * @return string|null
+     * @return string
```

Comment:
There is a check `$this->entityClass === null` line 84.

Are you sure this value can't be null ?

Reply:
Right, I'll revert this, what is wrong here is the property's phpdoc, let's fix it.